### PR TITLE
chore: update debian instructions to remove deprecated command

### PR DESCRIPTION
**Pull request by Piccirello**
Supersedes #355.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -77,10 +77,10 @@ $ apk upgrade doppler
 $ apt-get update && apt-get install -y apt-transport-https ca-certificates curl gnupg sudo
 
 # add Doppler's GPG key
-$ curl -sLf --retry 3 --tlsv1.2 --proto "=https" 'https://packages.doppler.com/public/cli/gpg.DE2A7741A397C129.key' | sudo apt-key add -
+$ curl -sLf --retry 3 --tlsv1.2 --proto "=https" 'https://packages.doppler.com/public/cli/gpg.DE2A7741A397C129.key' | gpg --dearmor | sudo dd status=none of=/etc/apt/keyrings/doppler.gpg
```
Comment (nmanoogian) [inline INSTALL.md]: curious: Why use `dd` here over file redirection?

**Review by nmanoogian — DISMISSED** [review]

File: INSTALL.md (hunk 1 above)
Comment (Piccirello) [reply INSTALL.md]: I switched away from `tee` here due to `gpg --dearmor` printing gobbledygook that made this command seem broken (when it is in fact correct). It also doesn't print a newline at the end of its output. I don't love the use of a command that folks might not be used to (`dd status=none ...`) but it seemed preferable to the alternative.

```sh
root@6c9ec0075e71:/# curl -sLf --retry 3 --tlsv1.2 --proto "=https" 'https://packages.doppler.com/public/cli/gpg.DE2A7741A397C129.key' | gpg --dearmor | sudo tee /etc/apt/keyrings/doppler.gpg
4$???'jz?|p??TeR?j?>??;?&???a?/??9?\8?6pL????KÅ??~??`???
                                                        T?h.8?j&?$a5#m?o???5q?盨yn
                                                                                  N?&r8?,U???'OQ?dG??iJ???k?>???`?v?#?=t]o(?[m???=KY?(U?r5??z
                                                                                                                                             ???6??t*?g-???m??	L??g?t*T$v?3:?F?[y?w????t?ȶ?@
?-?nI0??~H =`?:??R??????ӧ?/??????????a ???<???f??(j?^!5??!^??&??Al??????1?#?A???^"???B????X??民v????? ??$?g??jH?l?5N?P??&?^??;F`?\????O6??\e4????
U?k??z?r@??Ҋ)EZ???Qj)???wįsY!?c?????Ţ???d??????jABֆg??ڡ?Tr?p???#??????3DopplerHQ - Packages <support+packages@doppler.com>?>!??Q? t??Nn;?*wA???)`y?	Υ?
                                                                                                                                                          	
	
       ?
	?*wA???)Q??r???sfH_S??T
]?ɼ??R??⚃`'1??????Z?jz?XCd?|?(??1?O?.`g?/??X?\?ў'[t??r???֝?TM^| ?í,?u>???????3?
q???????????4C?_?[?%?'?Fժ????\n??
(T?l???_?S????W??G?H?,
                      3?,??D????M?6?J??\?2r?-a<O"?]?y??i???????
[?2?Zi???????cj?-N??t?P{+(C$)=??
                                ?.[?uQ???(^&?s#??????
                                                     U?5?o???m??F?&?,???zs?T
`y??c?L~hIPܴ??S???+]????-3??0F??^??[??;??0oK?_??P?????n???j?zd`??V?B?-?????3??,Phȯ?5k?D?8@?y6Օ??????,>?°??j??X?鮔TT??V???V5?qvFe???!`?????g*?/?$?w???9֫c!?????_d?\T?ю_3?e?5? ??^?LBd?ɛ*>)?P?lrM? ҉??1n?SІ?q?ɳ[? ?I?sw?? ?T?ѻ?L+?q???;???iۜX!CcD6?ڜ??M?????8M?m$?b[َN??L???Gy?̂d?????{????C?󼕻???N?Z{N??Qx?<0??֊???KJ?(ɢ?
                                                                                             I$?o1?JK???|?eb???G?ݪ??O?[??Z|????Ѡ?s????)?
                                                                                                                                        ݫ????]???˒i???????]M????ܰ?k.???FT??
                                                                                                                                                                          ??,?? =??W)y*?D?~L#?A?n??????Y?&!??Q? t??Nn;?*wA???)`y?
            	Υ?
	?*wA???)?f?d????zE?@&j-$?s?? F?z']uV???s?
????N*?q???
           ?hY????<	?CN2$?p??^.?l#????I????N?xx?&2??h?B~?w?S?x??x??~ow?????8??V?*	`???ĸ?t?Ũ?4Xŏ琺xHZB??7)?	???Z?]?*?'HZF?;&?]p?X??s?:~?Y#????????!Y$v?
 ?
  ?W??UK?[?sҴ5j
??˹z?????}???o}ð?
                ??'D
                    ???
                       <??????3?<k{M1?:L???9?b
/3"҆~???f5/0V?ɞu?ys???Q
??Z??-?xcI??w?ݏ&????? B(mfÄx?M7??9?#YOܥ?x??'v4??|:ʅ?N??g?????3???<?????~?б??=???D?]??*T???㭯/ň%????5??7????{i^???
            ?@root@6c9ec0075e71:/# 
```

Comment (nmanoogian) [reply INSTALL.md]: Ah, that makes sense. Piping to `sudo tee /etc/apt/keyrings/doppler.gpg > /dev/null` would probably work too but I don't find it much more readable than what you have here.

Comment (Piccirello) [reply INSTALL.md]: I do like that better. I've switched back to `tee`.

**Review by nmanoogian — APPROVED** [review]

**Pull request merged**